providers/software_sources.rb

```diff
@@ -18,7 +18,15 @@
       
       current_lists = []
       remote_lists = []    
-      remote_lists.push("gecosv2.list")
+      
+      case $gecos_os
+      when "GECOS V3"
+        remote_lists.push("official-package-repositories.list")
```

Comment:
In GECOS V3 there is an application called "software sources" that helps a user to restore the original Linux repositiories and to add PPAs and other external software repositories.
This "software-sources" application belongs to a package called "mintsources" that is a dependence of "mint-meta-core" package, that is a dependence of "mint-meta-cinnamon", that is a dependence of "cinnamon-desktop-environment". So it's installed when you install cinnamon.

The "official-package-repositories.list" file is created by "software-sources".